bolts/fair/data/datamodules/tabular/compas.py

```diff
@@ -9,6 +14,40 @@
 class CompasDataModule(TabularDataModule):
     """COMPAS Dataset."""
 
+    @parsable
+    def __init__(
+        self,
+        sens_feat: Literal["Sex", "Race", "Race-Sex"] = "Sex",
```

Comment:
`Literal` isn't "parsable" by hydra, right?

Reply:
Yes, this needs to be an enum